src/sentry/release_health/metrics.py

```diff
@@ -220,7 +221,7 @@ def _get_crash_free_rate_data(
             where=[
                 Condition(Column("org_id"), Op.EQ, org_id),
                 Condition(Column("project_id"), Op.IN, project_ids),
-                Condition(Column("metric_id"), Op.EQ, metric_id(org_id, "session")),
+                Condition(Column("metric_id"), Op.EQ, metric_id(org_id, MetricKey.SESSION)),
```

Comment:
Shouldn't this be `MetricKey.SESSION.value` ? I see that the function definition takes a `str`

Reply:
Definitely, thanks!

Reply:
Gotchya!